src/coreclr/gc/gc.cpp

```diff
@@ -36767,6 +36765,9 @@ size_t gc_heap::desired_new_allocation (dynamic_data* dd,
         float     f = 0;
         size_t    max_size = dd_max_size (dd);
         size_t    new_allocation = 0;
+        float     time_since_previous_collection_secs = (dd_time_clock (dd) - dd_previous_time_clock (dd))*1e-6f;
+
+
```

Comment:
Nit: remove one empty line?

Reply:
Looking at the code, I think I should remove both of the empty lines - no reason to separate the declarations here, I think.